pkgs/by-name/pr/previous/package.nix

```
@@ -0,0 +1,71 @@
+{
+  lib,
+  stdenv,
+  fetchsvn,
+  cmake,
+  ninja,
+  SDL2,
+  libpcap,
+  libpng,
+  zlib,
+  readline,
+  copyDesktopItems,
+  makeDesktopItem,
+}:
+
+let
+  revision = "1667";
+in
+stdenv.mkDerivation {
+  pname = "previous";
+  version = "3.9-unstable-2025-09-02";
```

[Revision 1667](https://sourceforge.net/p/previous/code/1667/) was done on 2025-07-21

```suggestion
  version = "3.9-unstable-2025-07-21";
```